<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordChangeDialogClass</class>
 <widget class="QDialog" name="passwordChangeDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/crystal_project/password.png</normaloff>:/icons/crystal_project/password.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentEdit">
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>New password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newEdit">
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Retype new password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="changeButton">
     <property name="text">
      <string>Change</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/crystal_project/password.png</normaloff>:/icons/crystal_project/password.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
